pkg/apis/config/feature_flags.go

```diff
@@ -202,14 +227,16 @@ func NewFeatureFlagsFromMap(cfgMap map[string]string) (*FeatureFlags, error) {
 	// defeat the purpose of having a single shared gate for all alpha features.
 	if tc.EnableAPIFields == AlphaAPIFields {
 		tc.EnableTektonOCIBundles = true
-		tc.EnableSpire = true
-	} else {
-		if err := setFeature(enableTektonOCIBundles, DefaultEnableTektonOciBundles, &tc.EnableTektonOCIBundles); err != nil {
+		// Only consider SPIRE if alpha is on.
```

Comment:
should we consider it an error if someone tries to configure SPIRE when alpha is off?

Reply:
Maybe, my earlier thought was adding an error will just throw a message for an impossible flow.
But if we do not then user will not know why SPIRE was not enabled in this flow even if the flag was set.
So added an error here.